lib/cocoapods-downloader/base.rb

```diff
@@ -121,6 +121,18 @@ def checkout_options
         raise 'Abstract method'
       end
 
+      # Returns a User-Agent string that itentifies http network requests as
+      # originating from CocoaPods.
+      #
+      # @param  [String] version
+      #         the current cocoapods version, defaulted to Pod::VERSION.
+      #
+      # @return [String] the User-Agent string
+      #
+      def self.user_agent_string(version = Pod::VERSION)
```

Comment:
I dont think `Pod::VERSION` is actually defined in this gem

Reply:
It is not.  I've been trying to get a feel for the independence of this Gem from the CocoaPods gem, and have updated to a safer access of `Pod::VERSION` to try to provide the expected general-case version without the access concern.

Expanding a bit beyond the feature request, I added the the cocoapods-downloader gem version to the User-Agent, so that it looks like: `CocoaPods/1.9.1 cocoapods-downloader/1.3.0`.  If this is not preferred, I'm happy to revert.

Reply:
Nice! seems worthwhile to include that